auth/auth-router.js

```diff
@@ -1,11 +1,59 @@
 const router = require('express').Router();
+const bcryptjs = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+
+const configs = require("../api/configs");
+const Users = require("./auth-model");
+const { isValid } = require("./auth-services");
 
 router.post('/register', (req, res) => {
-  // implement registration
+  const credentials = req.body;
+  if(isValid(credentials)){
+    const rounds = process.env.BCRYPT_ROUNDS || 4;
+    const hash = bcryptjs.hashSync(credentials.password, rounds);
+    credentials.password = hash;
+    Users.add(credentials)
+      .then(user => {
+        res.status(201).json(user)
```

Comment:
Everywhere else, you return an object with a `message` and some data. I would do the same here `{ message: 'User created successfuly', user }`. In general, standardizing the way you give data to the user is good.